### PR TITLE
State facet 554

### PR DESCRIPTION
**Pull request by mccalluc**
Put state facet as headings in the org facet. If we were to do it separately
- It would be less clear what "state" is all about.
- The orgs list is harder to read. (As it has been.)

I want the facet add/remove behavior to be better, but this isn't intolerable.

### CODE REVIEW

**Comment by mccalluc** [comment]
(Towards #554.)

**Comment by mccalluc** [comment]
Folks supported this idea. Link to state facet, but maybe exclude the state facet when counting stations?

**Comment by mccalluc** [comment]
@afred: I think this is what we want: Clicking on the state brings in all the orgs under that state. (In the back of my mind, I'm wondering now if the links from the org pages should work the same way, but someone else can file a bug if they are so lead.)

File: lib/aapb.rb [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+    'VT' => 'Vermont',
+    'VA' => 'Virginia',
+    'WA' => 'Washington',
+    'WV' => 'West Virginia',
+    'WI' => 'Wisconsin',
+    'WY' => 'Wyoming'
+  }
+  STATES.default = 'undefined' 
+  # Without a default we get a cryptic error message
+  # about comparing array to array for the sort.
```
Comment (afred) [inline lib/aapb.rb]: @mccalluc does this cryptic error come from ruby or javascript?

Comment (mccalluc) [inline lib/aapb.rb]: Ruby, but this will be moot, because this list is going to go away again with the NAEB stuff: I'm adding a state.rb and states.yml.

**Pull request merged**